src/Contract/Collection.php

```diff
@@ -361,10 +353,8 @@ interface Collection extends
     Tailable,
     Tailsable,
     TakeWhileable,
-    Timesable,
```

Comment:
hmm why remove these from the interface?

Reply:
Because these are static constructors and I think they should belong to the concrete implementation only.